### PR TITLE
docs (Controller.md): Add new feat description

### PR DESCRIPTION
**Pull request by Maledong**
Due to https://github.com/eggjs/egg-multipart/pull/19, we have to
add this new feature into Egg's main doc in controller.md to notify
clients.

- [x] `npm test` passes
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

### CODE REVIEW

**Comment by fengmk2** [comment]
感谢！

**Comment by Maledong** [comment]
Thanks!

File: docs/source/zh-cn/basics/controller.md [hunk 1]
```diff
@@ -314,17 +314,113 @@ module.exports = {
 
 ### 获取上传的文件
 
-请求 body 除了可以带参数之外，还可以发送文件，一般来说，浏览器上都是通过 `Multipart/form-data` 格式发送文件的，框架通过内置 [Multipart](https://github.com/eggjs/egg-multipart) 插件来支持获取用户上传的文件。
+请求 body 除了可以带参数之外，还可以发送文件。一般来说，浏览器上都是通过 `multipart/form-data` 格式发送文件的，在 Egg 的 Multipart 插件中，我们为你提供了两种方式上传和接受文件：
 
-完整的上传示例参见：[eggjs/examples/multipart](https://github.com/eggjs/examples/tree/master/multipart)。
+- #### File 模式：
+如果你完全不知道 Nodejs 中的 Stream 用法，那么 File 模式非常合适你：
```
[the remark this answers is not in the document]
Comment (Maledong) [reply docs/source/zh-cn/basics/controller.md]: 这是中文的逗号，后面无需认为加空格的吧？

File: docs/source/zh-cn/basics/controller.md [hunk 2]
```diff
@@ -314,17 +314,113 @@ module.exports = {
 
 ### 获取上传的文件
 
-请求 body 除了可以带参数之外，还可以发送文件，一般来说，浏览器上都是通过 `Multipart/form-data` 格式发送文件的，框架通过内置 [Multipart](https://github.com/eggjs/egg-multipart) 插件来支持获取用户上传的文件。
+请求 body 除了可以带参数之外，还可以发送文件。一般来说，浏览器上都是通过 `multipart/form-data` 格式发送文件的，在 Egg 的 Multipart 插件中，我们为你提供了两种方式上传和接受文件：
```
[the remark this answers is not in the document]
Comment (Maledong) [reply docs/source/zh-cn/basics/controller.md]: 好的，我会提一个PR修正，谢谢你。

**Pull request merged**